data-prepper-core/src/integrationTest/resources/data_prepper.yml

```diff
@@ -0,0 +1 @@
+useTls: false
```

Comment:
nit: For keeping it consistent with the source configuration and elasticsearch configs, can we call this as `ssl:false`